apple-pie/index.html

```diff
@@ -4,18 +4,54 @@
     <meta charset="UTF-8" />
     <meta name="viewport" content="width=device-width, initial-scale=1.0" />
     <title>Apple Pie Recipe</title>
-    <!-- don't forget to link your styles -->
+    <link rel="stylesheet" href="styles/style.css">
   </head>
+  <header>
+    <img src="/home/patrick/Schreibtisch/IronHack/lab-css-recipes-clone/apple-pie/images/apple-pie.jpg" alt="apple pie" id="background">
```

Comment:
src references like this will break on every other machine! 
It's relative to your exact file system and nobody else's will look like this. 
Try using relative imports instead. src="images/apple-pie.jpg"